### DISCUSSION TITLE
[Feature Request] Suggest KB Article in reply

### DISCUSSION THREAD

**Discussion by caygri** [osTicket/osTicket#6229] [Ideas]
Hello,

please suggest to link kb article or import article in agent reply.

**Comment by ntozier** [comment]
Is this a [Feature Request]? (please add that to title)

You can make canned responses with the links in them.

You can paste a URL into the response.

**Comment by caygri** [comment]
Hello, yes this is a features request like freshdesk or zendesk.


![Screenshot of  #1428  Huge updates coming SOON to Hey Oliver AppSumo plan _ Caygri Supporto](https://user-images.githubusercontent.com/5012957/186114852-7e397919-d47a-48ce-9725-539cfd70daf9.jpg)

I know the canned responses but if someone has big kb it's hard to link it. 
Canned reply is more used for say "hello or thank you"

It's for different uses case.
